### ISSUE TITLE
Bottom line of some post replies is cutoff

### ISSUE THREAD

**Issue by RealAMJB** [christianselig/apollo-bugs#1056]
**Describe the bug**
After updating to 1.10.3 from 1.8 I’ve noticed that the bottom row of comment text is sometimes cutoff when in landscape mode. If I scroll down to the point the comment is off screen and then back up it appears fine. 

While somewhat random for which comments it happens to, it is pretty reproducible by going to posts with lots of comments. 

The 1.10 announcement has several replies it happens to for example. 

**To Reproduce**
1. Open App
2. Click Post
3. Scroll through comments
4. See anomaly

**iOS version**: 13.3.1
**Apollo version**: 1.10.3
**iOS device**: 11 Pro Max
**Jailbroken**: No
**Reproducible:** Yes

**Expected behavior**
The bottom line of comment replies aren’t cutoff. 

**Screenshots**

https://reddit.com/r/apolloapp/comments/j0l0xh/post_text_cutoff_in_landscape_ios_13_apollo_1103/

**Comment by RealAMJB** [comment]
Appears to be resolved in 1.10.4.
